File: src/main/resources/output/IFC4X3_java/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcSurface/IfcBoundedSurface/IfcCurveBoundedSurface/IfcCurveBoundedSurface.java
public class IfcCurveBoundedSurface extends IfcBoundedSurface {
    public IfcSurface BasisSurface;
    public SET [1:?] OF IfcBoundaryCurve Boundaries;
    public IfcBoolean ImplicitOuter;
}

File: src/main/resources/output/IFC4X3_js/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcSurface/IfcBoundedSurface/IfcCurveBoundedSurface/IfcCurveBoundedSurface.js
class IfcCurveBoundedSurface extends IfcBoundedSurface {
    constructor() {
        /** @type {IFCSURFACE} */
        this.BasisSurface = null;
        /** @type {SET [1:?] OF IFCBOUNDARYCURVE} */
        this.Boundaries = null;
        /** @type {IFCBOOLEAN} */
        this.ImplicitOuter = null;
    }
}
